examples/scripts/minimal_example_of_lookup_tables.py

```diff
@@ -0,0 +1,51 @@
+import pybamm
+import pandas as pd
+import numpy as np
+
+
+def process_2D(name, data):
+    data = data.to_numpy()
+    x1 = np.unique(data[:, 0])
+    x2 = np.unique(data[:, 1])
+
+    value = data[:, 2]
+
+    x = (x1, x2)
+
+    value_data = value.reshape(len(x1), len(x2), order="C")
+
+    formatted_data = (name, (x, value_data))
+
+    return formatted_data
+
+
+parameter_values = pybamm.ParameterValues(pybamm.parameter_sets.Chen2020)
+
+# overwrite the diffusion coefficient with a 2D lookup table
+D_s_n = parameter_values["Negative electrode diffusivity [m2.s-1]"]
+df = pd.DataFrame(
+    {
+        "sto": [0, 1, 0, 1, 0, 1],
+        "T": [0, 0, 25, 25, 45, 45],
+        "D_s_n": [D_s_n, D_s_n, D_s_n, D_s_n, D_s_n, D_s_n],
+    }
+)
```

Comment:
is this the standard way you would provide this data (with repetition of sto and T)?

Reply:
Just noticed that the T column (which is changing slower) should actually be the first column so I will update this

I've seen this format used a bit for 2D lookup tables (also used it for the ECM lookup tables in PyBaMM); is there another format you use instead?

Reply:
I didn't have anything else in mind specifically, seems clunky to have to repeat entries but can be done easily enough with code